### PR TITLE
Update alt text for SVG calendar icon on past events

### PR DESCRIPTION
**Pull request by nick-mon1**
### Summary
Added proper alt text for calendar svg icon

### Problem
Calendar icon for past events does not contain an alternative text or label.

image

### Solution
Added an `id` to the `title` attribute of the svg and a `aria-labelledby` to the `svg`.

image

### CODE REVIEW

File: themes/digital.gov/layouts/events/card-event-past.html [hunk 1]
```diff
@@ -8,9 +8,9 @@
     </div>
     {{- else -}}
     <div class="non-youtube-event-card grid-col-12 tablet:grid-col-4 tablet:order-last grid-row">
-      <div aria-role="img" class="grid-row flex-align-self-center" data-link="{{ .Permalink }}">
-        <svg xmlns="http://www.w3.org/2000/svg" role="img" height="150" width="150" viewBox="0 0 25 25">
-          <title>Non-Youtube Event</title>
+      <div aria-role="img" alt="" class="grid-row flex-align-self-center" data-link="{{ .Permalink }}">
+        <svg xmlns="http://www.w3.org/2000/svg" role="img" aria-labelledby="calendar-title" height="150" width="150" viewBox="0 0 25 25">
+          <title id="calendar-title">Calendar Event</title>
```
Comment (mejiaj) [inline themes/digital.gov/layouts/events/card-event-past.html:13]: Not necessary because we've hidden it, but we can leave.

**Review by mejiaj — CHANGES_REQUESTED** [review]

File: themes/digital.gov/layouts/events/card-event-past.html [hunk 2]
```diff
@@ -8,9 +8,9 @@
     </div>
     {{- else -}}
     <div class="non-youtube-event-card grid-col-12 tablet:grid-col-4 tablet:order-last grid-row">
-      <div aria-role="img" class="grid-row flex-align-self-center" data-link="{{ .Permalink }}">
-        <svg xmlns="http://www.w3.org/2000/svg" role="img" height="150" width="150" viewBox="0 0 25 25">
-          <title>Non-Youtube Event</title>
+      <div aria-role="img" alt="" class="grid-row flex-align-self-center" data-link="{{ .Permalink }}">
```
[the remark this answers is not in the document]
Comment (nick-mon1) [reply themes/digital.gov/layouts/events/card-event-past.html:11]: Done

File: themes/digital.gov/layouts/events/card-event-past.html [hunk 3]
```diff
@@ -8,9 +8,9 @@
...
+        <svg xmlns="http://www.w3.org/2000/svg" role="img" aria-labelledby="calendar-title" height="150" width="150" viewBox="0 0 25 25">
```
[the remark this answers is not in the document]
Comment (nick-mon1) [reply themes/digital.gov/layouts/events/card-event-past.html:12]: Done

File: themes/digital.gov/layouts/events/card-event-past.html (hunk 1 above)
Comment (nick-mon1) [reply themes/digital.gov/layouts/events/card-event-past.html:13]: I have left this since it is a best practice but not used in this case.

**Pull request closed without merging**